B827EBFFFE5E11C5.json

```diff
@@ -0,0 +1,19 @@
+{ 
+    "gateway_conf": { 
+        "gateway_ID": "B827EBFFFE5E11C5", 
+        "servers": [ 
+            { 
+                "server_address": "ttn-router-us-west", 
```

Comment:
Please enter the full url here, check the list here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html